@types/svg-to-jsx.d.ts

```diff
@@ -0,0 +1,10 @@
+declare module "svg-to-jsx" {
```

Comment:
why is this folder called "@types"? why is it in root?

Reply:
can you add a comment about what this does and why this is here. i know it but it should be clearer

Reply:
I can rename to types

Reply:
would be better.
why is it not in source?